File: acmeshop/acmeshop/src/main/java/org/acme/shop/model/Product.java
package org.acme.shop.model;

import io.quarkus.hibernate.reactive.panache.PanacheEntity;

import jakarta.persistence.Entity;

@Entity
public class Product extends PanacheEntity {

    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: acmeshop/acmeshop-kotlin/src/main/kotlin/shop/model/Product.kt
package shop.model

import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity
import jakarta.persistence.Entity

@Entity
class Product : PanacheEntity() {

    companion object: PanacheCompanion<Product> {
    }

    lateinit var name: String

}